apps/rewards/app/app-logic.js

```diff
@@ -0,0 +1,65 @@
+import PropTypes from 'prop-types'
+import React, { useCallback, useMemo, } from 'react'
+import { AragonApi, usePath } from '@aragon/api-react'
+
+const PATH_REGEX = /^\/(rewards|myrewards)(?:\/([0-9])?)?(?:\/)?$/
+
+const TABS = [ 'Overview', 'My Rewards' ]
+
+const urlify = (str) => {
+  if(str) {
+    str = str.toLowerCase()
+    str = str.replace(' ', '')
+  }
+  return str
+}
+
+export const usePathSegments = () => {
+  const [ path, requestPath ] = usePath()
+
+  const fromPath = useMemo(() => {
+    const [ , urltab, rewardId ] = path.match(PATH_REGEX) || []
```

Comment:
We use to follow conventions and define variables with `camelCase` names
```suggestion
    const [ , urlTab, rewardId ] = path.match(PATH_REGEX) || []
```

Reply:
Fixed in b99b2aa2e43941637c74582a4a71ade4d8635147